src/Drawer/Drawer.js

```diff
@@ -246,7 +246,9 @@ class Drawer extends Component {
   onBodyTouchStart = (event) => {
     const swipeAreaWidth = this.props.swipeAreaWidth;
 
-    const touchStartX = event.touches[0].pageX;
+    const touchStartX = !this.context.muiTheme.isRtl ?
```

Comment:
@oliviertassinari Thanks for the review 🙏. Removed unnecessary negations from the code, please let me know if any other changes required.